src/stylesheets/index.css

```
@@ -42,9 +42,9 @@
   width: 298px;
 }
 
-.link {
-  @apply text-navy-700 hover:underline font-semibold;
-}
+/* .link {
+  @apply text-navy-700 hover: underline font-semibold;
+} */
```

```suggestion
.link {
  @apply text-navy-700 hover: underline font-semibold;
}
```

Unrelated change, please address in separate issue.